GOVERNANCE.md

```diff
@@ -0,0 +1,143 @@
+This process MUST be completed prior to the candidate being named an official Veraison maintainer.
+
+The onboarding period is intended to ensure that the to-be-appointed maintainer is able/willing to take on the time requirements, familiar with core logic and concepts, understands the overall system architecture and interactions that comprise it, and is able to work well with both the existing maintainers and the community.
+
+## Decision Making at the Veraison org level
+
+When maintainers need to make decisions there are two ways decisions are made, unless described elsewhere.
+
+The default decision making process is [lazy-consensus][lazy-consensus].
+This means that any decision is considered supported by the team making it as long as no one objects.
+Silence on any consensus decision is implicit agreement and equivalent to explicit agreement.
+Explicit agreement may be stated at will. In the case of security critical decisions more explicit consensus may be needed.
+
+When a consensus cannot be found a maintainer can call for a [majority][majority] vote on a decision.
+
+Many of the day-to-day project maintenance can be done by a lazy consensus model.
+But the following items must be called to vote:
+
+* Removing a maintainer for any reason other than inactivity (super majority)
+* Changing the governance rules (this document) (super majority)
+* Licensing and intellectual property changes (including new logos, wordmarks) (simple majority)
+* Adding, archiving, or removing sub-projects (simple majority)
+* Utilizing Veraison/CNCF money for anything CNCF deems "not cheap and easy" (simple majority)
+
+New sub-projects should be created (or added) with a well defined mission and goals, and significant changes should be voted on by both the sub-project maintainers and the org maintainers.
+
+Other decisions may, but do not need to be, called out and put up for decision via creating an [issue][issue] at any time and by anyone.
+By default, any decisions called to a vote will be for a _simple majority_ vote.
+
+Meetings should be publicly documented ([Zulip chat][zulip-chat]), and recorded and notes kept.
+Meetings should have a chair, this is a rotating role not restricted to maintainers.
+
+## Code of Conduct
+
+This Veraison project has adopted the [CNCF Code of Conduct][cncf-coc].
+
+## Attributions
+
+* This governance model was created using the [Notary Project](https://github.com/notaryproject/notary/blob/master/GOVERNANCE.md) governance documents.
+
+## DCO and Licenses
+
+The following licenses and contributor agreements will be used for Veraison projects:
+
+* [Apache 2.0](https://opensource.org/licenses/Apache-2.0) for code
+* [Developer Certificate of Origin](https://developercertificate.org/) for new contributions
+
+[cncf-coc]:             https://github.com/cncf/foundation/blob/master/code-of-conduct.md
```

Comment:
we should move https://github.com/veraison/veraison/blob/main/CODE_OF_CONDUCT.md in this repo.

Reply:
See: https://github.com/veraison/veraison/issues/130